terraform/modules/webservers/webservers.tf

```diff
@@ -72,6 +72,15 @@ resource "aws_security_group" "qa-reports-lb-sg" {
   }
 }
 
+data "aws_subnet_ids" "ourvpc" {
+  vpc_id = "${var.vpc_id}"
+}
+
+data "aws_subnet" "oursubnets" {
+  count = "${length(data.aws_subnet_ids.ourvpc.ids)}"
+  id = "${data.aws_subnet_ids.ourvpc.ids[count.index]}"
+}
+
```

Comment:
Right now our aws account has one VPC with two subnets, but that should not be assumed. Rather than trying to use all subnets in the account, this should use the list of subnets provided in values(availability_zone_to_subnet_map). In the future, other subnets may be added for other purposes, and qa-reports should not automatically try to use them.

Instead, try the following:
```
data "aws_subnet" "oursubnets2" {
  count = "${length(values(var.availability_zone_to_subnet_map))}"
  id = "${element(values(var.availability_zone_to_subnet_map), count.index)}"
}
```

Reply:
done